src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/Server/SqlNormalizer.cs

```diff
@@ -66,9 +69,12 @@ internal BinaryOrderedUdtNormalizer(Type t, bool isTopLevelUdt)
                 _isTopLevelUdt = true;
```

Comment:
I don't see any usage for `isTopLevelUdt` parameter in this function. Seems it'd been used in the `else` section. 🤔

Reply:
Yes. I left it there in the comments in case it's needed later.